Minio/DataModel/ObjectArgs.cs

```diff
@@ -23,16 +25,23 @@ public abstract class ObjectArgs<T> : BucketArgs<T>
 
     internal string ObjectName { get; set; }
     internal byte[] RequestBody { get; set; }
+    internal int ExactBodySize { get; set; }
 
     public T WithObject(string obj)
     {
         ObjectName = obj;
         return (T)this;
     }
 
-    public T WithRequestBody(byte[] data)
+    public T WithRequestBody(byte[] data, int exactBodySize = 0)
```

Comment:
Instead of using `exactBodySize` as a parameter, it would be more consistent to use `count` similar to functions like Buffer.BlockCopy.